pairing/bls12381/suite_test.go

```diff
@@ -0,0 +1,404 @@
+package bls12381_test
+
+import (
+	"bytes"
+	"crypto/cipher"
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.dedis.ch/kyber/v3"
+	this "go.dedis.ch/kyber/v3/pairing/bls12381"
+	"go.dedis.ch/kyber/v3/util/random"
+)
+
+// Code extracted from kyber/utils/test
+// TODO: expose API in forked drand/kyber
+// Apply a generic set of validation tests to a cryptographic Group,
+// using a given source of [pseudo-]randomness.
+//
+// Returns a log of the pseudorandom Points produced in the test,
+// for comparison across alternative implementations
+// that are supposed to be equivalent.
+func testGroup(t *testing.T, g kyber.Group, rand cipher.Stream) []kyber.Point {
```

Comment:
Why can't you run this on `G1`, `G2`, and `GT`? Because they don't implement `Embed`?